Make line item price explicit (#502)
centrapay/centrapay-docs: legacy/src/api/payment-requests/payment-requests.md

{% h4 Mandatory Fields %}

| Field |        Type        |                                                      Description                                                       |
| ----- | ------------------ | ---------------------------------------------------------------------------------------------------------------------- |
| name  | String             | The product description.                                                                                               |
| sku   | String             | The product (stock keeping unit) code.                                                                                 |
| qty   | {% dt BigNumber %} | The product quantity (eg. item count, weight, volume etc).                                                             |

> | price | {% dt BigNumber %} | The combined price of each individual product in cents (eg. price = individual product price * qty - discounts + tax). |

Reviewer: ```suggestion
| price | {% dt BigNumber %} | The total price in cents for the line item (eg. price = product price * qty - discounts + tax). |
```
